src/pages/Home/Home.tsx

```diff
@@ -1,13 +1,25 @@
+import { useState } from "react";
+import { RecipeType } from "../../shared/models/recipes.model";
+import { getRecipes } from "../../shared/services/recipesService";
 import Card from "../../ui/Card";
 import Container from "../../ui/Container";
 import LayoutGeneral from "../../ui/Layouts/LayoutGeneral";
 import Categories from "./Categories";
 
 const Home = () => {
+  const [recipes, setRecipes] = useState<Array<RecipeType>>([]);
+
+  const onClickCard = (category: string) => {
+    const data = getRecipes(category);
+    data.then((data) => {
+      setRecipes(data.data.meals);
+    });
+  };
+
   return (
     <LayoutGeneral
-      navbar={undefined}
-      categories={<Categories />}
+      navbar={"FOOD ROUTINE"}
```

Comment:
Maybe, you can add a component to show the header of the Home

Reply:
Yes, I understand you, it will be done in other PR.